src/api.js

```diff
@@ -0,0 +1,7 @@
+import axios from "axios";
+
+export const getReviews = () => {
+    return axios.get('https://dh-nc-games.herokuapp.com/api/reviews').then((response) => {
```

Comment:
Would be nice to use axios.create in this file to create a base url that all of your functions can utilise (fine as it is for now though, given that there's just the one function in there)